lib/matplotlib/tests/test_image.py

```diff
@@ -258,11 +258,11 @@ def test_image_composite_background():
     ax.set_axis_bgcolor((1, 0, 0, 0.5))
     ax.set_xlim([0, 12])
 
-@image_comparison(baseline_images=['image_composite_alpha'], remove_text=True)
-def test_image_composite_alpha():
```

Comment:
I was trying to make things consistent by using `combine_images` everywhere instead of `image_composite`.